### PR TITLE
Label Design System

### PR DESCRIPTION
**Pull request by JadhavJeet**

### CODE REVIEW

**Comment by cg-mmirca** [comment]
Great work Jeet the PR looks really good! 👏 👏 

I'll just make some small suggestions on selected lines if you don't mind.

**Comment by cg-mmirca** [comment]
You got all the key elements in place Jeet! :)

Probably only missing the themes stories?

File: stories/themes/accessible.theme.css [hunk 1]
```diff
@@ -6,3 +6,12 @@
...
   --dcx-font-formcontrol_error-line-height: 20px;
 }
+
+.dcx-label {
+  --dcx-border-radius-formcontrol: 0px;
+  --dcx-spacing-y-formcontrol: 30px;
+  --dcx-font-formcontrol_hint-size: 16px;
+  --dcx-font-formcontrol_hint-line-height: 20px;
+  --dcx-font-formcontrol_error-size: 16px;
+  --dcx-font-formcontrol_error-line-height: 20px;
```
Comment (cg-mmirca) [inline stories/themes/accessible.theme.css:16]: Would be good to check on these overwrites. Currently we only have the following tokens for the label styles in `label.css`

```
.dcx-label {
  color: token('color-text-formcontrol_label');
  font-weight: token('font-weight-formcontrol_label');
  font-size: token('font-size-formcontrol_label');

  &.dcx-error-message {
    color: token('color-text-formcontrol_label-error');
  }
}
```

We could remove the ones that don't show up in the stylesheet.

For the accessible theme actually there may be little we need to change. Maybe we could use a stronger shade of red for the error and remove the rest?

On this note, we could add a DarkTheme as well, we could use the white color text to show how the component looks on a black background.

**Comment by cg-mmirca** [comment]
Thank you again Jeet, everything looks good from my side :)

I just pushed some adjustments for the accessible and dark themes. Improving the color contrast against the background.

**Review by daniele-zurico — APPROVED** [review]

**Pull request merged**